AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSafeContentFilterTest.xml

```diff
@@ -28,6 +28,10 @@
             <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
             <actionGroup ref="logout" stepKey="logout"/>
         </after>
+        
+        <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForPear">
+            <argument name="query" value="cars"/>
+        </actionGroup>
         <grabTextFrom selector="{{AdobeStockSection.recordsFound}}" stepKey="countWithoutSafeFilter"/>
         <actionGroup ref="AdminAdobeStockExpandFiltersActionGroup" stepKey="expandFilters"/>
         <actionGroup ref="AdminUserApplySafeSearchFilterActionGroup" stepKey="see32imagesOnTheSecondPage"/>
```

Comment:
I know this is not your specific introduced change, but while we are here we may as well clean the test up. Can we update the `stepKey` here as well to correctly reflect the step?
```suggestion
        <actionGroup ref="AdminUserApplySafeSearchFilterActionGroup" stepKey="applySafeSearchFilter"/>
```